### ISSUE TITLE
All parts of site should have WIA-ARIA roles.

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/persona#95]
http://www.w3.org/WAI/intro/aria

**Comment by shane-tomlinson** [comment]
This will help with accessibility and is something that can be updated as we touch the HTML.

**Comment by shane-tomlinson** [comment]
Closing until we decide to tackle this.
